### PR TITLE
Improve performance of patient finder with tags on large DB

### PR DESCRIPTION
**Pull request by kchapple**
On a large DB, the join on the  tf_patient_tags pivot table caused big performance penalty, so I added a migration to insert an index to pid col of tf_patient_tags pivot table.

### CODE REVIEW

File: interface/tags_filters/migrations/Migration_2018_08_22_14_48_11.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+--    behavior:  If the index does not exist, it will be created
+
+--  #EndIf
+--    all blocks are terminated with and #EndIf statement.
+
+--
+-- Indexes for table `tf_patients_tags`
+--
+
+ALTER TABLE `tf_patients_tags`
```
Comment (kchapple) [inline interface/tags_filters/migrations/Migration_2018_08_22_14_48_11.php]: This is really the only code. Migrations are only run once. There is no need for conditional logic in this case.

File: interface/tags_filters/migrations/Migration_2018_08_22_14_48_11.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+--    arguments: table_name colname
+--    behavior:  If the index does not exist, it will be created
+
+--  #EndIf
+--    all blocks are terminated with and #EndIf statement.
+
+--
+-- Indexes for table `tf_patients_tags`
+--
+
```
Comment (kchapple) [inline interface/tags_filters/migrations/Migration_2018_08_22_14_48_11.php]: This is really the only code. Migrations are only run once. There is no need for conditional logic in this case.

Comment (aethelwulffe) [reply interface/tags_filters/migrations/Migration_2018_08_22_14_48_11.php]: Does the migration require any user input to run?

**Comment by kchapple** [comment]
No, these migrations are run automatically. Essentially, the first request made to the server, the migration will be run. The system keeps track of the migrations that are run so they are only run once. 

Ken
kenchapple.com
facebook.com/kenchapplemusic
@ken_chapple

Check out my latest CD, Crossing the Black here: https://open.spotify.com/album/1fM4v6hJ1275fywjaREzNP?si=ohHDH2vV

**Pull request merged**